zkevm/etherman/etherman.go

```diff
@@ -165,6 +165,9 @@ func NewClient(cfg Config) (*Client, error) {
 	case params.SepoliaChainConfig.ChainID.Uint64():
 		l1Conf = params.SepoliaChainConfig
 		l2Conf = params.ChainConfigByChainName(zkchainconfig.GetChainName(cfg.L2ChainID))
+	case params.HermezLocalDevnetChainConfig.ChainID.Uint64():
+		l1Conf = params.HermezLocalDevnetChainConfig
```

Comment:
can we make it configurable somehow? like l1-chain-id in the config files or something? because we still wanna run this against sepolia or mainnet for our tests

Reply:
Do you mean somethings like this?

```go
default:
    log.Debug("Custom L1 chain ID", "l1ChainID", cfg.L1ChainID)
    l1Conf = params.ChainConfigByChainName(zkchainconfig.GetChainName(cfg.L1ChainID))
    l2Conf = params.ChainConfigByChainName(zkchainconfig.GetChainName(cfg.L2ChainID))
```

Reply:
yep yep, something like that

Reply:
Implemented in [chore: make l1 config configurable](https://github.com/0xPolygonHermez/cdk-erigon/pull/190/commits/9e1f7ca341f6cbffcbe7eb8e36ccc9b7ac445000) :)